public/app/percona/dbaas/components/DBCluster/ResourcesBar/ResourcesBar.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({ colors, palette, spacing, typography }: GrafanaTheme) => ({
+  resourcesBarWrapper: css`
+    display: flex;
+    flex-direction: row;
+  `,
+  iconWrapper: css`
+    margin-right: ${spacing.sm};
+  `,
+  resourcesBarContent: css`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+  `,
+  resourcesBarBackground: css`
+    background-color: ${palette.gray1};
+    border-radius: 50px;
+    height: ${spacing.sm};
+    position: relative;
+    width: 100%;
+  `,
+  resourcesBarLabel: css`
+    color: ${colors.textSemiWeak};
+    font-size: ${typography.size.sm};
+    margin-top: ${spacing.xs};
+  `,
+  captionWrapper: css`
+    align-items: center;
+    display: flex;
+    margin-top: ${spacing.xs};
+  `,
+  captionSquare: css`
+    width: 10px;
+    height: 10px;
+  `,
+  allocatedSquare: css`
+    background-color: ${palette.blue80};
+  `,
+  expectedSquare: css`
+    background-color: #043464;
```

Comment:
This hex is used more than once for now. Could we refactor into a constant (not in the constants file necessarily, just at the top of this file) ?

Reply:
Done!